packages/lavapack/test/sourcemaps.spec.js

```diff
@@ -5,40 +5,40 @@ const test = require('ava')
 // const UglifyJS = require('uglify-js')
 // const { SourceMapConsumer } = require('source-map')
 
-test.skip('sourcemaps - adjust maps for wrapper', async (t) => {
-  const { wrapIntoModuleInitializer } = require('../src/sourcemaps')
-  const fooSource = `
-  var two = 1 + 1
-  throw new Error('Boom')
-  var three = two + 1
-  `
+// test.skip('sourcemaps - adjust maps for wrapper', async (t) => {
+//   const { wrapIntoModuleInitializer } = require('../src/sourcemaps')
```

Comment:
this `require()` and any other `require()` in this file was bad.  I could also just `rm` the file. 😄